Scenarios/SelfHostedGitHubRunner.md

```diff
@@ -16,15 +31,25 @@ GitHub runners can be registered for an organization (accessible for all reposit
 1. Select **Review and Create** and then review the deployment and choose **Create**.
 1. Wait for the Azure VM creation to finalize, navigate back to see that the Runners have been registered and are ready to use.
 ![Runners](https://github.com/microsoft/AL-Go/assets/10775043/ba90e239-a8ee-4297-8bed-a30e3fc3db8a)
+
+## Allow your repository access to your runners
 1. On the list of Runners on GitHub, choose the runner group **Default** and allow public repositories if your repository is public.
 ![public](https://github.com/microsoft/AL-Go/assets/10775043/9bdd01ab-ac67-44bf-bfd1-af5c5ec91364)
 1. Now navigate to your project settings file (.AL-Go/settings.json) and set **gitHubRunner** to **self-hosted**.
+   - Note that you can use other tags than **self-hosted** to identify special runners for GitHub jobs, or you can set gitHubRunner to "self-hosted, Windows" to ensure that a Windows version is selected if you have self-hosted linux runners as well.
 1. Save and inspect your workflows performance increase on the second run.
 1. Inspect that one of the runners pick up the workflow.
 ![Active](https://github.com/microsoft/AL-Go/assets/10775043/dfcd369c-ad54-427e-92d4-153afda30b53)
 1. Clicking the runner reveals that the job it is running.
 ![Job](https://github.com/microsoft/AL-Go/assets/10775043/0ae30c22-9352-4864-a80e-81ed4ecd93e1)
 
+## GitHubRunner vs. runs-on
+You might have noticed that there are two settings in the repository settings file controlling which runners are selected, [GitHubRunner](https://aka.ms/algosettings#githubrunner) and [Runs-On](https://aka.ms/algosettings#runs-on).
+
+`"runs-on"` is used for all jobs, which doesn't build/test your app. This is jobs that doesn't require a lot of machine power, jobs which doesn't require a docker container. AL-Go for GitHub supports using ubuntu-latest for runs-on, which are faster and cheaper than windows-latest (which is the default). There are many jobs running in parallel using these runners and we do not recommend using self-hosted runners for this type of jobs.
+
+`"githubRunner"` is used for build/test jobs. This is jobs, which requires more memory, jobs which require a container. We do recommend self-hosted runners for these jobs if you want to increase build performance.
```

Comment:
```suggestion
`"githubRunner"` is used for build/test jobs, which require more memory and a container. 
```

Regarding the build increased performance; Doesn't it depend on what hardware you run your job on? I would just leave that sentence out.